ReactWindows/ReactNative/Views/TextInput/ReactTextInputShadowNode.cs

```diff
@@ -60,6 +64,42 @@ public ReactTextInputShadowNode()
         }
 
         /// <summary>
+        /// Sets the background color for the <see cref="ReactTextBox"/>.
+        /// </summary>
+        /// <param name="color">The masked color value.</param>
+        [ReactProp(ViewProps.BackgroundColor, CustomType = "Color")]
+        public void SetBackgroundColor(uint? color)
```

Comment:
Background color is a prop that does not affect measurement, so it doesn't need to be in the shadow node.